demos/k8s-demo/01_security_admin_steps

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+. ./utils.sh
+. ./admin_config.sh
+
+##################################################
+step "Create a new namespace"
+
+delete_ns_and_cleanup "quick-start-backend-ns"
+kubectl create namespace "quick-start-backend-ns"
+
+##################################################
+step "Add certificates to Kubernetes Secrets"
+
+# add pg certificates to kubernetes secrets
+kubectl --namespace quick-start-backend-ns \
+  create secret generic \
+  quick-start-backend-certs \
+  --from-file "etc/pg_server.crt" \
+  --from-file "etc/pg_server.key"
+
+##################################################
+step "Create StatefulSet for Database"
+
+kubectl --namespace quick-start-backend-ns apply \
+  --filename "etc/pg.yml"
+
+wait_for_app "quick-start-backend" "quick-start-backend-ns"
+# why doesn't this work?
+# kubectl --namespace quick-start-backend-ns \
+#   rollout status "statefulset/quick-start-backend"
+
+##################################################
+step "Create Application Database"
+
+kubectl --namespace quick-start-backend-ns \
+ exec --stdin "$(first_pod quick-start-backend quick-start-backend-ns)" \
+ -- \
+  psql --username "${DB_ADMIN_USER}" \
+  --command "CREATE DATABASE quick_start_db;"
+
+##################################################
+step "Create Database Table and Permissions"
+
+docker run --rm --interactive \
+ --env PGPASSWORD="${DB_ADMIN_PASSWORD}" \
+ "postgres:9.6" \
+    psql \
+    --username "${DB_ADMIN_USER}" \
+    "postgres://$DB_URL" \
+    <<EOSQL
+/* Create Application User */
+CREATE USER ${DB_USER} PASSWORD '${DB_INITIAL_PASSWORD}';
+
+/* Create Table */
+CREATE TABLE pets (
+    id serial primary key,
+    name varchar(256)
+);
+
+/* Grant Permissions */
+GRANT SELECT, INSERT ON public.pets TO ${DB_USER};
+GRANT USAGE, SELECT ON SEQUENCE public.pets_id_seq TO ${DB_USER};
+EOSQL
+
+##################################################
+step "Store DB credentials in Kubernetes Secrets"
+
+delete_ns_and_cleanup "quick-start-application-ns"
+kubectl create namespace "quick-start-application-ns"
```

Comment:
maybe worth a comment about switching to a ns for the app?

also, I was reading about k8s recently and I'm not sure our namespace-happy patterns are typical. I [read](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) that custom namespaces aren't even recommended if your team is less than ~10 people:
> For clusters with a few to tens of users, you should not need to create or think about namespaces at all

so this isn't something we've thought about before, but I think it might be more typical to have the backend in the same ns as the app.

in any case, I think it merits a comment here.